tests/unittests/aggregation_tests.py

```
@@ -1615,6 +1616,512 @@ def test_expr(self):
+            foc._PARALLEL_MAX_PARTITIONS = max_partitions
+            foc._PARALLEL_SAMPLES_PER_PARTITION = 10
+            return asyncio.run(view._async_aggregate(aggregations))
+        finally:
+            foc._PARALLEL_MAX_PARTITIONS = orig
+            foc._PARALLEL_SAMPLES_PER_PARTITION = orig_spp
+
+    def _assert_results_equal(self, view, aggregations):
+        single = self._run_async_agg(view, aggregations, 1)
+        multi = self._run_async_agg(view, aggregations, 8)
+        self.assertEqual(len(single), len(multi))
+        for i, (s, m) in enumerate(zip(single, multi)):
```

Ruff (0.15.9)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)